### DISCUSSION TITLE
how i can use this project in next js

### DISCUSSION THREAD

**Discussion by vahidalvandi** [GrapesJS/grapesjs#5680] [Q&A]
i want to use this project in next js please help me how to i cant build it in after edit

**Comment by mosh-tudor** [comment]
@vahidalvandi 

The code in Grapesjs is already "built" (because you have plain HTML and CSS).
You just need to find a way to insert ^ into the page you want.

https://nextjs.org/docs/app/building-your-application/data-fetching
